python/pyarrow/array.pxi

```diff
@@ -3977,12 +3977,12 @@ cdef class StructArray(Array):
 
     @staticmethod
     def from_arrays(arrays, names=None, fields=None, mask=None,
-                    memory_pool=None):
+                    memory_pool=None, type=None):
         """
         Construct StructArray from collection of arrays representing
         each field in the struct.
 
-        Either field names or field instances must be passed.
+        Either field names, field instances or a structtype must be passed.
```

Comment:
Could we use struct type with a space or `StructType`?

Reply:
@AlenkaF I went with struct type to be consistent with the existing message.

Reply:
Ah, I see this is changed in the last commit. Thanks!